### PR TITLE
chore: Update npm dependencies

### PR DESCRIPTION
**Pull request by soGit**
### Description

Update webapp dependencies

### CODE REVIEW

File: webapp/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ In a terminal, start frontend GUI server using NG :
 
 ```shell
 $ cd src/main/webapp
-$ npm install --force
```
Comment (lbroudoux) [inline webapp/README.md:46]: Could you explain the differen between those 2 optionsN

File: webapp/src/main/webapp/package.json [hunk 2]
```diff
@@ -12,47 +12,47 @@
...
-    "@angular/common": "~8.2.0",
-    "@angular/compiler": "~8.2.0",
-    "@angular/core": "~8.2.0",
-    "@angular/forms": "~8.2.0",
-    "@angular/platform-browser": "~8.2.0",
-    "@angular/platform-browser-dynamic": "~8.2.0",
-    "@angular/router": "~8.2.0",
-    "jquery": "^3.5.0",
-    "keycloak-js": "^20.0.2",
+    "@angular/animations": "~8.2.14",
```
Comment (lbroudoux) [inline webapp/src/main/webapp/package.json:15]: These ones seem reasonable updates. Did you get the opportunity to test them out?

File: webapp/README.md (hunk 1 above)
Comment (soGit) [reply webapp/README.md:46]: --legacy-peer-deps is about ignoring peer dependencies version conflicts, while --force is about ignoring a wider range of errors and checks. And here we just want to temporary ignore peer dependencies version checks (until project updated to the new components library)

File: webapp/src/main/webapp/package.json (hunk 2 above)
Comment (soGit) [reply webapp/src/main/webapp/package.json:15]: After update dependencies, I did a minimal manual check

**Review by lbroudoux — APPROVED** [review]
LGTM. Starting the checks/

**Pull request merged**